src/OTSession.js

```diff
@@ -17,7 +17,7 @@ export default class OTSession extends Component {
     const sessionEvents = sanitizeSessionEvents(this.props.eventHandlers);
     setNativeEvents(sessionEvents);
     this.createSession();
-    logOT(this.props.apiKey, this.props.sessionId);    
+    logOT(this.props.apiKey, this.props.sessionId, 'rn_initialize');
```

Comment:
I would prefer OTLog instead of logOT. This way you can package all OT* (sdk + your own)

Reply:
`OTLog` does not follow camel case and it's not global so I think it's fine as is.